.github/workflows/build_and_test.yml

```diff
@@ -53,10 +53,13 @@ jobs:
         run: ./scripts/check_for_blobs.sh
       - name: Build libafl debug
         run: cargo build -p libafl
-      - name: Test the book
+      - name: Test the book (Linux)
         # TODO: fix books test fail with updated windows-rs
-        if: runner.os != 'Windows'
+        if: runner.os == 'Linux'
         run: cd docs && mdbook test -L ../target/debug/deps
+      - name: Test the book (MacOS)
+        if: runner.os == 'MacOS'
+        run: cd docs && mdbook test -L ../target/debug/deps -L /Library/Frameworks/Python.framework/Versions/3.13/lib
```

Comment:
This will fail once 3.14 is out...

Reply:
Do you know how to link python in a version agnostic manner?

Reply:
Maybe something like
```bash
3.$(python --version | cut -d' ' -f2 | cut -d'.' -f2)
```

Reply:
Ah actually the best way is `python-config --ldflags`